webviz_config/_theme_class.py

```diff
@@ -75,6 +75,38 @@ def adjust_csp(self, dictionary: dict, append: bool = True) -> None:
             else:
                 self._csp[key] = value
 
+    def create_themed_layout(self, layout: dict) -> dict:
+        """
+        Create a new Plotly layout dict by merging the input layout with the theme layout,
+		prioritizing the input kayout if there are conflicts with the theme layout. In addition:
+		For the special case of multiple xaxes or yaxes, E.g. xaxis2 and xaxis3 (for a secondary
+		and tertiary xaxis), will get the theme xaxis/yaxis layout, unless they are defined
+        themselves as e.g. xaxis2 in the theme layout. (Note that e.g. xaxis2 still needs to be
+		in the input layout, just not in the theme layout.
+        """
```

Comment:
Hmm, yes, seems like regular tabs instead of 4 spaces had found there way in somehow 🤔 Surprised black/pylint didn't catch something like that. Fixed now.